freqtrade/freqtradebot.py

```diff
@@ -547,11 +549,37 @@ def _notify_buy(self, trade: Trade, order_type: str) -> None:
             'type': RPCMessageType.BUY_NOTIFICATION,
             'exchange': self.exchange.name.capitalize(),
             'pair': trade.pair,
-            'limit': trade.open_rate,
+            'limit': trade.open_rate_requested,
```

Comment:
note to myself: verify this change separately

Reply:
I'll fix it, thanks for feedback.

Reply:
Fixed.